### PR TITLE
feat: publish roadmap page

### PR DESCRIPTION
**Pull request by Keith-CY**
Preview: https://nervos-official-website-git-publish-roadmap-page-magickbase.vercel.app/roadmap

### CODE REVIEW

**Comment by Keith-CY** [comment]
#### 📝 Changed routes:


| [**/roadmap/Journey.module**](https://nervos-official-website-5ux4jbbjw-magickbase.vercel.app/roadmap/Journey.module) | [**/roadmap/Journey**](https://nervos-official-website-5ux4jbbjw-magickbase.vercel.app/roadmap/Journey) |
|:-:|:-:|
|<a href="https://nervos-official-website-5ux4jbbjw-magickbase.vercel.app/roadmap/Journey.module">Screenshot of /roadmap/Journey.module</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/bQ0CdDGNW0io4pUYIdLRAG-G0RhWCqcUWHxx6OkMSfCf4TSzO4CtEt8pyS_LPq0JSkjH5GwmDYtpfMQEjUEIAg.png">(view full size)</a> |<a href="https://nervos-official-website-5ux4jbbjw-magickbase.vercel.app/roadmap/Journey">Screenshot of /roadmap/Journey</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/AvYAntic10cgVAeAs_Zg6eEDPE7jJSQE5QiSYRmVExpvVX_HrDTaSVEMCLy9hNClDXHErJ9vFHx3onznDiea7A.png">(view full size)</a> |



| [**/roadmap/Section.module**](https://nervos-official-website-5ux4jbbjw-magickbase.vercel.app/roadmap/Section.module) | [**/roadmap/Section**](https://nervos-official-website-5ux4jbbjw-magickbase.vercel.app/roadmap/Section) |
|:-:|:-:|
|<a href="https://nervos-official-website-5ux4jbbjw-magickbase.vercel.app/roadmap/Section.module">Screenshot of /roadmap/Section.module</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/kbO9l6tx9BgcuUdyOPY9fBlRWhZHoaBHBkAf5_8wMc5LA5rBVmqYC9lZf-WTfoTciSApHn6vUsir5utYXIOwXQ.png">(view full size)</a> |<a href="https://nervos-official-website-5ux4jbbjw-magickbase.vercel.app/roadmap/Section">Screenshot of /roadmap/Section</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/u-_esDjzSa1BDpr1PG71uOOiWVcBjbovKj2hbO3r2XeO6j4oydw8bR9AhqaKmKf7I2UNaipliKph5fXfeV8MxQ.png">(view full size)</a> |



| [**/roadmap/index.page**](https://nervos-official-website-5ux4jbbjw-magickbase.vercel.app/roadmap/index.page) |
|:-:|
|<a href="https://nervos-official-website-5ux4jbbjw-magickbase.vercel.app/roadmap/index.page">Screenshot of /roadmap/index.page</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/JtIHcBz8uckAp0kW_u_W3uOZ0ces-xKpDeNo2MfC4rU-P2aU8z6YE47CQxqKi6aNFxynYtoJsoKkSOrFP390kA.png">(view full size)</a> |







Commit 1959c39c59767f0caf699fb825453eb89f815f72 (https://nervos-official-website-5ux4jbbjw-magickbase.vercel.app).

File: src/pages/roadmap/Section.tsx [hunk 1]
```diff
@@ -29,32 +28,29 @@ export const Section: React.FC<SectionProps> = ({
...
-        <div className={styles.descriptionText}>
-          {descriptionText} {/* TODO: add real href here */}
-          <StyledLink className={styles.link} href={''} colored>
-            {descriptionLink}
-          </StyledLink>
-        </div>
-        {projects.map(project => (
-          <div className={styles.project} key={project}>
-            {project}
+        {descriptionText.startsWith('Lorem') ? null : (
```
Comment (WhiteMinds) [inline src/pages/roadmap/Section.tsx:42]: This unique business logic implementation is a bit hidden, making it difficult for people to quickly discover its relationship with locales. Consider using more obvious markers:

1. "//description": "..."
1. "description": "\_\_hidden_this_description\_\_ ..."

File: src/pages/roadmap/Section.module.scss [hunk 2]
```diff
@@ -10,6 +10,24 @@
 
 .icon {
   transform: translate(-140px, -20px);
+
+  img {
+    @media screen and (max-width: 1080px) {
+      width: 244px !important;
```
[the remark this answers is not in the document]
Comment (Keith-CY) [reply src/pages/roadmap/Section.module.scss]: `!important` was removed by using `max-width/max-height` in commit https://github.com/Magickbase/nervos-official-website/pull/360/commits/dbed03c83cd71b44d9911903cf8d48c717a69db2

File: src/pages/roadmap/Section.tsx (hunk 1 above)
Comment (Keith-CY) [reply src/pages/roadmap/Section.tsx]: Prefix `__hidden__description__` was used as an annotation in commit https://github.com/Magickbase/nervos-official-website/pull/360/commits/c673c32c11668cc5d46c6eda893ba0feaf78c6f4

**Review by WhiteMinds — APPROVED** [review]

**Pull request merged**